application.py

```diff
@@ -1135,8 +1143,11 @@ def delete_contest(contest_id):
 
     db.execute("BEGIN")
     db.execute("DELETE FROM contests WHERE id=:cid", cid=contest_id)
-    db.execute("DROP TABLE :cid", cid=contest_id)
-    db.execute("DROP TABLE :cidinfo", cidinfo=contest_id + "info")
+
+    db.execute("DELETE FROM contest_users WHERE contest_id=:cid", cid=contest_id)
+    db.execute("DELETE FROM contest_solved WHERE contest_id=:cid", cid=contest_id)
+    db.execute("DELETE FROM contest_problems WHERE contest_id=:cid", cid=contest_id)
+
     db.execute("COMMIT")
     os.remove('metadata')
```

Comment:
Final review: No idea where this came from but it should not be here

Reply:
This is when the contest is deleted (the contest tables are dropped). Before, the code was:

```python
db.execute("DROP TABLE :cid", cid=contest_id)	
db.execute("DROP TABLE :cidinfo", cidinfo=contest_id + "info")
```

Reply:
no i'm talking about `os.remove('metadata')`

Reply:
I have no idea, it was in the repository when I forked it.

Reply:
I have removed it.